plugins/mcreator-localization/lang/texts.properties

```diff
@@ -47,16 +47,55 @@ action.gradle.clear_caches.option.message=<html><b>This action will clear all Gr
   build of every generator type and version will take a bit longer the first time.<br>\
   In some cases, clearing entire Gradle folder is needed to resolve Gradle errors.<br><br>\
   <small>Any running build tasks or open test clients will be closed during this action.
+action.common.cancel=Cancel
+action.common.apply=Apply
+action.common.create=Create
+action.common.resize=Resize
 action.gradle.clear_caches.option.title=Clear all Gradle caches
 action.gradle.close_mcreator_while_installation_message=<html><b>Gradle is still setting up the workspace!</b><br>You can not close MCreator while the Gradle is setting up. Wait until the setup is complete.<br>You can monitor the progress of the setup in the Console tab.
 action.gradle.close_mcreator_while_installation_title=Gradle setting up
 action.gradle.close_mcreator_while_running_message=<html><b>There are some Gradle tasks running in the background.</b><br>Are you sure you want to close this workspace?<br><small><br>If you proceed, running tasks will be canceled!
 action.gradle.close_mcreator_while_running_title=Close workspace
+dialog.common.base_color=Base color:

```

Comment:
Since keys in this section here are used only in the image editor dialogues, you can use the dialog.imageeditor key instead of dialog.common.

Reply:
Done!